### DISCUSSION TITLE
Pagination component style props are not working as expected

### DISCUSSION THREAD

**Discussion by saifobeidat** [quasarframework/quasar#9854] [General - Components / Directives / etc]
I am trying to use flat prop to remove the numbers background, but it seems not working as expected

Here's a snippet form Quasar's site
```      
<q-pagination
        v-model="current"
        :max="5"
        direction-links
        flat
        color="grey-6"
        active-color="primary"
      />
```

And here is the result of the above snippet:
![image](https://user-images.githubusercontent.com/24841143/123753871-f922e200-d8c2-11eb-8aa6-8d2542620fec.png)

It has no effect..

I am using `@quasar/extras": "^1.10.5`, `quasar": "^1.15.16`..

**Comment by MilosPaunovic** [comment]
Hello @saifobeidat,

Could you provide a codepen for this issue, because I tried and could not reproduce?
